starter-core/src/test/groovy/io/micronaut/starter/feature/graalvm/GraalVMSpec.groovy

```diff
@@ -118,23 +118,6 @@ class GraalVMSpec extends ApplicationContextSpec implements CommandOutputFixture
         e.message == 'GraalVM is not supported in Groovy applications'
     }
 
-    @Unroll
-    void 'it is not possible to use graalvm with JDK versions different than JDK8 through JDK11'(JdkVersion jdkVersion) {
-        when:
```

Comment:
If you rollback the change I mentioned before about not supporting Java 16 with GraalVM yet, then you need to bring this test back.

Reply:
I still see this tests removed